src/harness/reference_models/pre_iap_filtering/pre_iap_filtering.py

```diff
@@ -36,16 +38,20 @@ def preIapReferenceModel(protected_entities, sas_uut_fad, sas_test_harness_fads)
       from SAS test harnesses.
   """    
 
-
   # Invoke Inter SAS duplicate grant purge list reference model
-  inter_sas_duplicate_grant.interSasDuplicateGrantPurgeReferenceModel\
-                                               (sas_uut_fad, sas_test_harness_fads)
+  inter_sas_duplicate_grant.interSasDuplicateGrantPurgeReferenceModel(sas_uut_fad,
+                                                          sas_test_harness_fads)
 
-  # TODO
   # Invoke PPA, EXZ, GWPZ, and FSS+GWBL purge list reference models
+  list_of_fss_neighboring_gwbl = pre_iap_util.getFssNeighboringGwbl(protected_entities['gwblRecords'],
+                                     protected_entities['fssRecords'], FSS_GWBL_PROTECTION_DISTANCE)
+  ppa_gwpz_fss_gwbl_purge.ppaGwpzFssPlusGwblPurgeReferenceModel(sas_uut_fad,
+      sas_test_harness_fads, protected_entities['ppaRecords'], 
+      protected_entities['palRecords'], protected_entities['gwpzRecords'], 
+      list_of_fss_neighboring_gwbl) 
 
   # Invoke FSS purge list reference model
   if 'fssRecords' in protected_entities:
-    fss_purge.fssPurgeReferenceModel(sas_uut_fad, sas_test_harness_fads,\
+    fss_purge.fssPurgeReferenceModel(sas_uut_fad, sas_test_harness_fads,
```

Comment:
Does the fss purge will be called in other context than the pre_iap_filtering?
If not, the fss_purge module should also be part of that directory. 
No need to multiply directories with only one file per directory.

Reply:
Agreed,   we will move the fss_purge module also into 'pre_iap_filtering', since fss   purge is with #PR462, will do the changes in respective PR

Reply:
Thanks for the move.